spec/support/devise.rb

```diff
@@ -0,0 +1,3 @@
+module DeviseSignInHelpers
```

Comment:
blast from a past, right? 😄 feel free to remove this one :)

Reply:
bump - still needs removal

Reply:
Check :)